test/shopify.test.js

```diff
@@ -529,9 +529,158 @@ describe('Shopify', () => {
         expect(res).to.deep.equal(data);
       });
     });
+
+    const addWorkingRESTRequestMock = () => {
+      nock(`https://${shopName}.myshopify.com`)
+        .get('/admin/shop.json')
+        .reply(200, () => ({
+          shop: {
+            name: 'My Cool Test Shop',
+            id: '1'
+          }
+        }));
+    };
+
+    const shopifyRetryableInstance = new Shopify({
+      accessToken,
+      parseJson,
+      shopName,
+      stringifyJson,
+      maxRetries: 3
+    });
+
+    it('should retry 429 errors from Shopify after the retry-after header has elapsed', async () => {
```

Comment:
Remove "should" and use third person present tense: `it('retries 429...'` and try to make it fit in an 80 chars line. It would also be nice to be consistent with other tests and use `async...await` only if it greatly simplifies the tests.